charts/admission-controller/templates/webhook/certificate.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.webhook.ssl.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    {{- include "admissionController.webhook.labels" . | nindent 4 }}
+  name: {{ include "admissionController.webhook.fullname" . }}
+  namespace: {{ include "admissionController.namespace" . }}
+spec:
```

Comment:
There's a lot we _could_ parameterize here, but I figure we should start simple and can always iterate.